docker/build.sh

```diff
@@ -77,4 +79,12 @@ CmakeBuild() {
   fi
 }
 
-[[ $SOURCED == "false" ]] && main "$@" || echo "Sourced!"
+if [[ $SOURCED == "false" ]]; then
+  # It is important to return exit code of main
+  # To be future-proof, this is handled explicitely
+  main "$@"
+  BUILD_RESULT=$?
+  exit BUILD_RESULT
```

Comment:
I think we need to access the variable explicitly
```suggestion
  exit $BUILD_RESULT
```

Reply:
Argh, yes. I just can't bash...